h/static/scripts/annotation-ui.js

```diff
@@ -141,6 +141,27 @@ function findByTag(annotations, tag) {
   });
 }
 
+/**
+ * Initialize the status flags and properties of a new annotation.
+ */
+function initializeAnnot(annotation) {
+  if (annotation.id) {
+    return annotation;
+  }
+
+  // Currently the user ID, permissions and group of new annotations are
+  // initialized in the <annotation> component controller because the session
+  // state and focused group are not stored in the Redux store. Once they are,
+  // that initialization should be moved here.
+
+  return Object.assign({}, annotation, {
+    // Copy $$tag explicitly because it is non-enumerable
+    $$tag: annotation.$$tag,
```

Comment:
This is obviously a bit silly. I intend to refactor the way local tags are assigned to annotations to avoid the need for this, and probably also change it from `$$tag` to just `$tag`. That is outside the scope of this PR though.